### PR TITLE
Split 3054 minor changes

### PR DESCRIPTION
**Pull request by polybassa**
Minor fixes to the current code. Could this PR please get merged previously to #3283 and #3054

### CODE REVIEW

**Comment by polybassa** [comment]
@gpotter2 Can this PR get merged, please?

File: test/contrib/automotive/interface_mockup.py [hunk 1]
```diff
@@ -192,15 +205,15 @@ def exit_if_no_isotp_module():
...
     import importlib
     if "scapy.contrib.isotp" in sys.modules:
-        importlib.reload(scapy.contrib.isotp)
+        importlib.reload(scapy.contrib.isotp)  # type: ignore
 
 load_contrib("isotp", globals_dict=globals())
 
 if six.PY3 and ISOTP_KERNEL_MODULE_AVAILABLE:
-    if not ISOTPSocket == ISOTPNativeSocket:
+    if not ISOTPSocket == ISOTPNativeSocket:  # type: ignore
```
Comment (gpotter2) [inline test/contrib/automotive/interface_mockup.py:213]: You should be using the `is` operator

File: test/contrib/automotive/interface_mockup.py [hunk 2]
```diff
@@ -209,3 +222,79 @@ def exit_if_no_isotp_module():
...
+            raise
+        except Exception:
+            if conf.debug_dissector:
+                from scapy.sendrecv import debug
+                debug.crashed_on = (cls, val)
+                raise
+            pkt = conf.raw_layer(val)
+        if ts:
+            pkt.time = ts
+        return pkt
```
Comment (gpotter2) [inline test/contrib/automotive/interface_mockup.py:290]: This function is a duplicate. Import it

File: test/contrib/automotive/interface_mockup.py [hunk 3]
```diff
@@ -209,3 +222,79 @@ def exit_if_no_isotp_module():
...
+
+    def sr1(self, *args, **kargs):
+        # type: (Any, Any) -> Optional[Packet]
+        from scapy import sendrecv
+        ans = sendrecv.sndrcv(self, *args, **kargs)[0]  # type: SndRcvList
+        if len(ans) > 0:
+            pkt = ans[0][1]  # type: Packet
+            return pkt
+        else:
+            return None
```
Comment (gpotter2) [inline test/contrib/automotive/interface_mockup.py:300]: same?

File: test/contrib/automotive/gmlan_trace.candump [hunk 4]
```diff
@@ -1,1087 +0,0 @@
-(0.297000) vcan2 12A#122010FF00001080
```
Comment (gpotter2) [inline test/contrib/automotive/gmlan_trace.candump:1]: It's a bummer this even got into here in the first place.. I'll try to be much more careful reviewing stuff from now on

**Review by gpotter2 — CHANGES_REQUESTED** [review]

File: test/contrib/automotive/interface_mockup.py (hunk 1 above)
Comment (polybassa) [reply test/contrib/automotive/interface_mockup.py]: Done

File: test/contrib/automotive/interface_mockup.py (hunk 2 above)
Comment (polybassa) [reply test/contrib/automotive/interface_mockup.py]: Done

File: test/contrib/automotive/interface_mockup.py (hunk 3 above)
Comment (polybassa) [reply test/contrib/automotive/interface_mockup.py]: Done

**Comment by polybassa** [comment]
Feedback applied

**Comment by polybassa** [comment]
@gpotter2 can this PR get merged, please?

**Review by gpotter2 — APPROVED** [review]

**Pull request merged**